pkg/gatewayserver/io/ws/ws.go

```diff
@@ -304,7 +305,16 @@ func (s *srv) handleTraffic(w http.ResponseWriter, r *http.Request) (err error)
 					return err
 				}
 			case down := <-conn.Down():
-				concentratorTime, ok := conn.TimeFromTimestampTime(down.GetScheduled().Timestamp)
```

Comment:
Okay this was indeed the issue. `Timestamp` is not set for absolute timed downlink.